app-drawer-layout/app-drawer-layout.html

```diff
@@ -207,10 +207,12 @@
        * @property drawer
        */
       get drawer() {
-        return Polymer.dom(this).querySelector('[slot=drawer]');
+        return Polymer.Element ?
+            Polymer.dom(this.$.drawerSlot).assignedNodes({ flatten: true })[0] :
+            Polymer.dom(this.$.drawerContent).getDistributedNodes()[0];
```

Comment:
Yeah, it doesn't work in Shady DOM... let's do something similar to this for now: https://github.com/PolymerElements/iron-dropdown/blob/2.0-preview/iron-dropdown.html#L350